guides/asynchronous-tasks/readme.md

```diff
@@ -359,44 +382,46 @@ The purpose of transient tasks is when a task is an implementation detail of an
 - A background worker or batch processing job which is independent of any specific operation, and is lazily created.
 - A cache system which needs periodic expiration / revalidation of data/values.
 
-Bearing in mind, in all of the above cases, you may need to validate that the background task hasn't been stopped, e.g.
+Here is an example that keeps a cache of the current time string since that has only 1-second granularity
+and you could be handling 1000s of requests per second.
+The task doing the updating in the background is an implementation detail, so it is marked as `transient`.
 
 ```ruby
 require 'async'
 require 'thread/local' # thread-local gem.
 
-class TimeCache
+class TimeStringCache
 	extend Thread::Local # defines `instance` class method that lazy-creates a separate instance per thread
 	
 	def initialize
-		@current_time = nil
+		@current_time_string = nil
 	end
 	
 	def current_time_string
 		refresh!
 		
-		return @current_time
+		return @current_time_string
 	end
 	
 	private
 	
 	def refresh!
 		@refresh ||= Async(transient: true) do
 			loop do
-				@current_time = Time.now.to_s
+				@current_time_string = Time.now.to_s
 				sleep(1)
 			end
 		ensure
-			# When the reactor terminates all tasks, this will be invoked:
+			# When the reactor terminates all tasks, this will be invoked.
```

Comment:
@ioquatix So this `ensure` block will be run because the reactor terminating the task will cause an exception to be raised from `sleep(1)`?

BTW do we prefer `sleep(1)` or `sleep 1` in this doc? I see both used here.

Reply:
> So this ensure block will be run because the reactor terminating the task will cause an exception to be raised from sleep(1)?

Yes that's correct.

> do we prefer sleep(1) or sleep 1

I probably prefer `sleep(1)`.

Also, when comments refer to the next line, I prefer to end with a `:`.

Reply:
Ok, I've updated all uses of `sleep` to `sleep(...)` and terminated the comment (on the next line) with `:`.